storage/plugins/book-club/src/LendingController.php

```
@@ -0,0 +1,286 @@
+
+        if ($this->lending->hasOpenOffer((int) $clubBook['id'], $userId)) {
+            $this->flash('error', __('Hai già un\'offerta aperta per questo libro.'));
+            return $this->redirect($response, $this->lendingPath($slug));
+        }
+
+        $notes = self::str($body, 'notes', 500);
+        if ($this->lending->createOffer((int) $club['id'], (int) $clubBook['id'], $userId, $notes) !== null) {
+            $this->flash('success', __('Copia offerta al club.'));
+        } else {
+            $this->flash('error', __('Impossibile completare l\'operazione. Riprova.'));
+        }
```

PHPMD (2.15.0)

[error] The method offer uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)